Makefile

```
@@ -108,8 +113,9 @@ tools: ## Install tools
 	$(GOCMD) install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION)
 	$(GOCMD) install google.golang.org/protobuf/cmd/protoc-gen-go
 
+client-python: python-client python-sdk
 
-client-python: api/swagger.yml  ## Generate SDK for Python client
+python-client: api/swagger.yml  ## Generate SDK for Python client - openapi generator version 5.3.0
```

Prefer to keep a common prefix for all SDKs, like we have common prefixes for other things here.
```suggestion
sdk-python-legacy: api/swagger.yml  ## Generate SDK for Python client - openapi generator version 5.3.0
```